examples/src/main/r/2-data-manipulation.R

```diff
@@ -0,0 +1,62 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+source("1-data.R")
```

Comment:
I'm not sure the `source` will work correctly given the way we encourage users to run example programs. If you see the `Running examples` section at http://people.apache.org/~pwendell/spark-releases/latest/index.html you can see that we ask users to run example programs with something like `./bin/spark-submit examples/src/main/r/data-manipulation.R`. 

So I'd just recommend merging all the code into one file `data-manipulation.R` and then take in the CSV file name as a command line argument.